docs/direct_publish_setup.md

```diff
@@ -0,0 +1,26 @@
+# Publishing an extension to OpenVSX on your own
+
+It is hightly advised that extension authors publish their extensions to OpenVSX as a part of their CI/CD process. See [our reasoning why that is](https://github.com/open-vsx/publish-extensions#when-to-add-an-extension).
+
+We have a simple template for setting a workflow with GitHub Actions which can help you kickstart your own. This template includes some neat features like:
+
+- Publishing to GitHub Releases, the Microsoft Marketplace and OpenVSX
+- Uploading the `.vsix` package to GitHub Releases as assets
+- Manual triggers that allow you to select what places to release to
```

Comment:
```suggestion
- Manual CI triggers for Open VSX and/or Microsoft Marketplace publishing.
```

Reply:
@loujaybee I think this new wording does not mention GitHub Releases, which are also a possibility. I am not sure how to rephrase it to still have the `and/or` part in it with three possible options 😄

Reply:
Updated! Does it work now? 😁

Reply:
I think this will work, although it does not mention GitHub Releases explicitly, but it is stated two points above, so it isn't all that necessary.